edb/schema/delta.py

```diff
@@ -3530,10 +3530,17 @@ def _cmd_from_ast(
         astnode: qlast.DDLOperation,
         context: CommandContext,
     ) -> RenameObject[so.Object_T]:
+        from . import modules
+
         parent_ctx = context.current()
         parent_op = parent_ctx.op
         assert isinstance(parent_op, ObjectCommand)
         parent_class = parent_op.get_schema_metaclass()
+        if parent_class == modules.Module:
```

Comment:
I think a cleaner approach would be to create a `RenameModule` implementation in `modules.py` and have it override `apply` to raise the error.

Reply:
Right now, renaming in general seems to be done via `AlterX` with a `Rename` subcommand. I think adding a `RenameModule` would be a significant deviation from the current implementation.

Reply:
It's always a `RenameFoo` subcommand. We have one for everything that supports renaming, though frequently it has no class body.